app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -18,23 +18,6 @@ module WorksControllerBehavior
       rescue_from WorkflowAuthorizationException, with: :render_unavailable
     end
 
-    class_methods do
-      def cancan_resource_class
```

Comment:
This method isn't in `ResourceController`.  Should it be?

Reply:
No, we don't need this anymore.